types/mercadopago/index.d.ts

```diff
@@ -0,0 +1,34 @@
+// Type definitions for mercadopago 1.5
+// Project: https://github.com/mercadopago/dx-nodejs
+// Definitions by: Daniel Pereira <https://github.com/danieldspx>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 3.5
+
+import { MercadoPagoConfig } from './configuration';
+import { MercadoPagoCard } from './resources/cards';
+import { MercadoPagoCustomer } from './resources/customers';
+import { MercadoPagoMerchantOrder } from './resources/merchantOrders';
+import { MercadoPagoPayment } from './resources/payment';
+import { MercadoPagoPreApproval } from './resources/preapproval';
+import { MercadoPagoPreference } from './resources/preferences';
+
+declare namespace MercadoPago {
+  const configure: MercadoPagoConfig['configure'];
+  const utils: any;
+  const configurations: MercadoPagoConfig;
+  const payment: MercadoPagoPayment;
+  const preferences: MercadoPagoPreference;
+  const preapproval: MercadoPagoPreApproval;
+  const merchant_orders: MercadoPagoMerchantOrder;
+  const customers: MercadoPagoCustomer;
+  const ipn: any;
+  const connect: any;
+  const money_requests: any;
+  const card: MercadoPagoCard;
+  const card_token: any;
+  const refund: any;
+  const discount_campaign: any;
+}
+
+// tslint:disable-next-line export-just-namespace
```

Comment:
This needs to be removed and error needs to be fixed

Reply:
Sorry but this is the only way to make things work tha way they should. If I follow https://github.com/Microsoft/dtslint/blob/master/docs/export-just-namespace.md we would go back to the initial commit:
https://github.com/danieldspx/DefinitelyTyped/blob/60ca22d56d5d9fbfe33c0001fdc7d1dc2d698ae2/types/mercadopago/index.d.ts#L14
And then we would have an error telling us to use `export =` and the only way is with the namespace. So this is a cyclic problem. The only way I have found to fix is disabling `export-just-namespace`. I described it in a comment before: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/49745#discussion_r529051802

I have looked typescript docs and all hints and I have no clue how to make this work withoud disabling it. Can you help?

Reply:
You need to convert all the properties of the namespace into into interface properties and define a const of interface type  and then export = that const

Reply:
Yep, this works well and seems pretty correct. I tryed doing this before but I was tryingo to assign it like `declare const mercadopago: MercadoPago = {}` and did not realize that I could just create it. Now i am commiting the final solution I guess :eyes: